### ISSUE TITLE
Responding to blog on flow rules to direct users to services

### ISSUE THREAD

**Issue by hutchwilco** [zerotier/ZeroTierOne#2270]
I'm writing this here at the invitation/suggestion on the blog post https://www.zerotier.com/blog/using-flow-rules-to-direct-users-to-services/

First of all: really appreciate the tutorial style of situation:context:code:explanation-of-code format. This helps a reader quickly onboard, and then go back and further understand the rules in a real use case.

I have a few issues - with some mistakes/typos in the blog post itself and the actual example rules not working as described:

1.  the identities used in the rules code, are human readable hostnames names like "deadbeef01", however Zerotier rules didn't like me using the Friendly name of a node, (I got an error unless I used the alphanumeric string identity eg 5CBB36347B) - which is not how the example is written.
2. The section from "Let arp, ping to servers work" onwards is missing content/text/screenshots. I don't know what's missing but it's hard to follow and makes no sense.
3. - My issue with the actual code - I followed the instructions, and lost access so I could not access the specified destination at port 22. I realise this is a trouble shooting issue I can go to the forum to resolve, but it's worth mentioning that this did work as described (and in the context of 1. above). This was after updating to the latest version 1.12.2 on both nodes.
To summarise: I expected to follow the instructions and be able to limit internode comms to just accessing a specific destination node, on specified ports. Instead I couldn't access the node at all.
To reproduce: create a zt network with two nodes, and use the instructions to limit access to just some services. You will instead have no access at all.

## Please let us know

- What you expect to be happening.
- What is actually happening?
- Any steps to reproduce the error.
- Any relevant console output or screenshots.
- What operating system and ZeroTier version. Please try the latest ZeroTier release.

**Comment by glimberg** [comment]
> 1. the identities used in the rules code, are human readable hostnames names like "deadbeef01",

These are, in fact, identities.  Not host names.  `deadbeef01` is a 10 digit hexadecimal string, which is the short text representation of a string.

> 2. The section from "Let arp, ping to servers work" onwards is missing content/text/screenshots. I don't know what's missing but it's hard to follow and makes no sense.

Yeah, I don't think that's supposed to be a section title.  That should be a code block.  Something must have gotten lost in translation on the last website upgrade.  I'll have a designer take a look.  That 2nd example with CAPs should be:

```
cap dns id 10 accept ipprotocol udp; accept ipprotocol tcp; accept dport 53; ;
cap http id 11 accept dport 80 or dport 443 and ipprotocol tcp; ;
cap htpc id 12 accept dport 80 or dport 32400 and ipprotocol tcp; ;
cap admin id 42 accept;

drop not chr ipauth;

# Let arp, ping to servers work.
accept ethertype arp or ipprotocol icmp4;

accept dport 53;

# accept;
break; # not drop, so we can use capabilities.
```

**Comment by glimberg** [comment]
Fixed the formatting on the blog post.  Further discussion is probably best for the forums over at https://discuss.zerotier.com

**Comment by hutchwilco** [comment]
thanks @glimberg!
The formatting with code blocks is much better!

Following up on:
> > the identities used in the rules code, are human readable hostnames names like "deadbeef01",
> These are, in fact, identities. Not host names. deadbeef01 is a 10 digit hexadecimal string, which is the short text representation of an identity. Same with cafebead01 and feedcafe01.

This is still not clear to me from the blog post and your description what an "identity" is.
The blog post says:
> To get these identities, one can zerotier-cli info from the CLI on the host, or check inside ZeroTier Central:
then shows the screenshot (which has two pieces that could be the identity - if one knows not to consider the MAC)
![image](https://github.com/zerotier/ZeroTierOne/assets/4792731/740da5d1-e84d-4852-8685-b63391bc3007)

There are two strings in the screenshot that could be the identity:
"59d9fd9396" which is called an address in ZeroTier Central.
or
"freebsd" which is the name given by the controller ( and throws an error if I try to use this in my rules)

is deadbeef01 only used in the blog post example as a nice human readable stand-in for the address 59d9fd9396 for the purposes of the example? Or am I missing something where an identity is being generated elsewhere but I can't see it in my ZT Central for some reason?
 
Sorry if I seem dense, but I think it needs clarifying for the post to be maximally helpful.

**Comment by glimberg** [comment]
Yes. They are just example identities
